src/audio/pipeline/pipeline-stream.c

```diff
@@ -486,6 +486,7 @@ static int pipeline_comp_trigger(struct comp_dev *current,
 			 * Initialization delay is only used with SSP, where we
 			 * don't use more than one DAI per copier
 			 */
+#if !CONFIG_LIBRARY
```

Comment:
Can we provide this API in the file module instead, even if it returns 0 today.

Reply:
Yep, I started to separate file component module adapter changes into a separate PR. It was quite simple to get dai_get_init_delay_ms() to pass without duplicating much of DAI data structures.